<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="20190729_115227" timestamp="2019-07-29T11:52:27" time="58.047" tests="36" errors="5">
  <testsuite id="1" name="secondAssignment" package="secondAssignment" hostname="ATMECSINDT-043" timestamp="2019-07-29T11:52:28" time="56.113" tests="36" errors="5">
    <properties>
      <property name="Run From" value="ATMECSINDT-043" />
      <property name="Run OS" value="amd64 Windows 10 10.0" />
      <property name="Run User" value="deepankar.pathal" />
      <property name="Iteration" value="1 of 1" />
      <property name="Script" value="secondAssignment.xlsx" />
      <property name="Logs" value="C:\projects\secondAssignment\output\20190729_115227\logs\nexial-20190729_115227.log" />
      <property name="Data File" value="secondAssignment.data.xlsx" />
      <property name="DataSheet(s)" value="#default" />
      <property name="JAVA_OPT" value="-Dorg.apache.poi.util.POILogger=org.apache.poi.util.NullLogger" />
      <property name="runtime args" value="-script C:\projects\secondAssignment\artifact\script\secondAssignment.xlsx" />
    </properties>
    <testcase id="Activity 1" name="Activity 1" classname="Scenario" timestamp="2019-07-29T11:52:30" time="54.129">
      <failure type="failure found in test activity" message="5 of 36 FAILED. Check C:\projects\secondAssignment\output\20190729_115227\secondAssignment.20190729_115228.001.xlsx for execution result, C:\projects\secondAssignment\output\20190729_115227\logs\nexial-20190729_115227.log for execution logs" />
      <system-out>Total : 36
Passed: 31
Failed: 5
Output: C:\projects\secondAssignment\output\20190729_115227\secondAssignment.20190729_115228.001.xlsx
Logs  : C:\projects\secondAssignment\output\20190729_115227\logs\nexial-20190729_115227.log</system-out>
    </testcase>
  </testsuite>
</testsuites>
